lighthouse-core/scripts/i18n/collect-strings.js

```diff
@@ -570,7 +570,7 @@ if (require.main === module) {
 
   if ((collisions) > 0) {
     console.log(`MEANING COLLISION: ${collisions} string(s) have the same content.`);
-    assert.equal(collisions, 16, `The number of duplicate strings have changed, update this assertion if that is expected, or reword strings. Collisions: ${collisionStrings}`);
+    assert.equal(collisions, 18, `The number of duplicate strings have changed, update this assertion if that is expected, or reword strings. Collisions: ${collisionStrings}`);
```

Comment:
@exterkamp is this bad?

Reply:
Yes. The new `columnHeader` "Element" is conflicting with `dom-size`'s "Element". You either need to write a meaning for both if they are semantically different. Or move it into `i18n.js`.

Reply:
We may need to update the assert for this to be more strongly worded+advice on what to do, because this is fatal for our translation pipeline if there are duplicates with no meanings to differentiate them.

Reply:
I see. Thanks @connorjclark for calling this out and thanks @exterkamp for the explanation. I've updated the PR accordingly and will avoid this in the future.